### ISSUE TITLE
Providing admin rights to a new account

### ISSUE THREAD

**Issue by rajikaimal** [scorelab/Bassa#123]
According to the current implementation, there is no way to assign **Admin** rights/privileges to a new account (from the user interface itself).

**Comment by saisankargochhayat** [comment]
That can be included in the setup procedure rather than in the application.

**Comment by rajikaimal** [comment]
@saisankargochhayat You mean in the initial project setup ? Oh yeah, it can be done too. But I was mentioning about an instance where new user is added and if that user needs to be granted with Admin rights. (This is after application setup)

**Comment by saisankargochhayat** [comment]
Yeah that's something we can add I guess . I guess we can all the
administrators to add more admins. :)

On 13 Apr 2017 11:48 a.m., "Rajika Imal" <notifications@github.com> wrote:

@saisankargochhayat <https://github.com/saisankargochhayat> You mean in the
initial project setup ? Oh yeah, it can be done too. But I was mentioning
about an instance where new user is added and if that user needs to be
granted with Admin rights. (This is after application setup)

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/scorelab/Bassa/issues/123#issuecomment-293794716>, or mute
the thread

**Comment by malithsen** [comment]
@rajikaimal This is a good suggestion :) We will also need an option to revoke privileges 
Do issue a PR when time permits :) 

@rehrumesh @saisankargochhayat
